util/passport.js

```diff
@@ -31,6 +36,73 @@ module.exports = function(passport) {
         });
     }));
 
+    passport.use(new GoogleStrategy({
+        clientID: config.google.clientID,
+        clientSecret: config.google.clientSecret,
+        callbackURL: config.baseURL + "/auth/google/callback"
+    }, function(req, accessToken, refreshToken, profile, done) {
+        function renderResponse(errorMsg) {
+            return responseFactory.sendRenderedResponse("public/signup", req, req.res, { captcha: app.recaptcha.render(), error: { message: errorMsg || "An unknown error occurred" }, username: "" });
```

Comment:
I meant to remove this... Next commit will get rid of this and make one function for all OAuths